### DISCUSSION TITLE
Can't use multiple simultaneous SMTP connections.

### DISCUSSION THREAD

**Discussion by gary1515** [postalserver/postal#2740] [Help with using Postal]
Can't use multiple simultaneous SMTP connections.

**Comment by gary1515** [comment]
I've encountered an issue where I have successfully set up a domain and email using SMTP in one software application, but I am experiencing difficulties when attempting to send emails through another software application that is also connected via SMTP.

**Comment by willpower232** [comment]
Have a look at #751 and #1926
